tests/python/contrib/test_hexagon/test_batch_matmul.py

```diff
@@ -0,0 +1,138 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Should we move these tests to an `topi/` subdirectory? Thinking about how we might want the directory structure. These are generic functionality tests, but soon we'll have tests for some of the same operators which utilize more complex scheduling. It would be nice to have these located in separate directories.

Reply:
makes sense. Moved them to sub-directories.